common/unsafe/src/main/java/org/apache/spark/unsafe/types/UTF8String.java

```diff
@@ -97,10 +99,10 @@ public static UTF8String fromBytes(byte[] bytes, int offset, int numBytes) {
   }
 
   /**
-   * Creates an UTF8String from given address (base and offset) and length.
+   * Creates an UTF8String from given memory block.
    */
-  public static UTF8String fromAddress(Object base, long offset, int numBytes) {
-    return new UTF8String(base, offset, numBytes);
+  public static UTF8String fromMemoryBlock(MemoryBlock base) {
```

Comment:
do we really need this? I think `new UTF8String(base)` is more concise than `UTF8String.fromMemryBlock`

Reply:
Good catch, let us drop this